hawkular-dmr-client/src/main/java/org/hawkular/dmr/api/OperationBuilder.java

```diff
@@ -529,7 +530,18 @@ public T resolveExpressions() {
 
         @SuppressWarnings("unchecked")
         public T resolveExpressions(boolean resolveExpressions) {
-            baseNode.get(ModelDescriptionConstants.RESOLVE_EXPRESSIONS).set(resolveExpressions);
```

Comment:
We should still set this because if this agent is also monitoring a remote EAP that happens to be EAP7 (which I think is still possible), this will be supported.

What we should do here is hardcode the string (don't use the constant, use the actual "resolve-expressions" string) but still do a check to see if the field exists so we can still log the warning. But the warning should say, "the local app server instance does not support resolving expressions").